layer-publisher/cdk.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Putting this comment here because the file above called is a binary and doesn't allow comments:

Do we need this `layer-publisher/cdk-lambda-powertools-python-layer@0.0.0.jsii.tgz`???

Reply:
<img width="1073" alt="image" src="https://user-images.githubusercontent.com/7353869/180485979-3ce5754c-01ce-4942-99ac-33b9423698bd.png">

Reply:
for now yes since my PR there is not merged yet : https://github.com/aws-samples/cdk-lambda-powertools-python-layer/pull/52

Reply:
Gotcha, thanks for the explanation